.github/workflows/build_hub.yml

```diff
@@ -141,5 +155,43 @@ jobs:
         run: |
           false
 
+  build_docker:
+    name: Build docker image
+    needs:
+      - unit_tests
+      - linting
+      - docs
+    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
+    runs-on: ubuntu-latest
+    env:
+      # These are for CI and not credentials of any system
+      POSTGRES_USER: prime
```

Comment:
You are duplicating the setting of these values.  I say define a variable early on and reuse it.